.github/workflows/commit-validation.yml

```diff
@@ -6,15 +6,49 @@
 name: "Commit message validation"
 on:
   pull_request:
-    branches: [main, commitlint]
-    paths: [
-      # Commit validation in PRs applies to build-tools only as of now.
-      'build-tools/**', '.github/workflows/commit-validation.yml']
+    branches: [main, next]
     types: [opened, synchronize, edited, ready_for_review]
 jobs:
+  # This job determines what paths have changed in the PR and outputs a boolean
+  changed-paths:
+    name: Determine changed paths
+    runs-on: ubuntu-latest
+    steps:
+      # For pull requests it's not necessary to checkout the code
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # ratchet:dorny/paths-filter@v2.11.1
+        id: filter
+        with:
+          filters: |
+            build-tools:
+              - 'build-tools/**'
+              - '.github/workflows/commit-validation.yml'
+    # Set job outputs to values from filter step
+    outputs:
+      build-tools: ${{ steps.filter.outputs.build-tools }}
+
+  # This job checks that PR template placeholder content has been removed from the PR body.
+  placeholder-content:
+    name: PR template placeholder content
+    runs-on: ubuntu-latest
+    steps:
+      - uses: sitezen/pr-comment-checker@f1e956fac00c6d1163d15841886ae80b7ae58ecb # ratchet:sitezen/pr-comment-checker@v1.0.1
+        with:
+          pr_description_should_not_contain: |
+            Feel free to remove or alter parts of this template that do not offer value for your specific change
+          wrong_pr_description_message: |
+            Your PR description contains placeholder content from the PR template. Remove or replace the placeholder
+            content. More information at:
+            https://github.com/microsoft/FluidFramework/wiki/Commit-message-style#pr-template-content
+
+  # We've adopted conventional commits in the build-tools release group. The CI jobs below require the PR title and body
+  # to match our standards. More information can be found in the wiki:
+  # https://github.com/microsoft/FluidFramework/wiki/Commit-message-style
   conventional-pr-title:
     name: PR title matches conventional commit format
     runs-on: ubuntu-latest
+    # Only runs if build-tools is changed
+    needs: changes
+    if: ${{ needs.changes.outputs.build-tools == 'true' }}
```

Comment:
I'm assuming the intention is to remove this once we start applying globally? Should we leave a TODO comment so we don't miss this? I could definitely see missing this when editing / removing filters from the initial job.

Reply:
Left without a TODO since the plans for the rest of the repo are unclear atm.